README.md

```diff
@@ -34,9 +34,15 @@ Please ensure you are using <b> python2.7. </b>
 
 Similar to Task 2, start the data feed server by running the python server
 
-<code> python2 datafeed/server.py </code>
+<code> python datafeed/server.py </code>
 
-Run <code>npm install</code> to start the React application.
+If you encounter an issue with `datautil.parser`, run this command: 
+
+	pip install python-dateutil
+
+If you don't have pip, you can install it from: https://pip.pypa.io/en/stable/installing/
+
+Run <code>npm install</code> to start the React application. Make sure to run your terminal/command prompt as administrator.
```

Comment:
let's also say that it's fine if there are audit warnings resulting after the npm install